webrtc.html

```diff
@@ -2373,18 +2376,26 @@ <h3>Interface Definition</h3>
   attribute EventHandler oniceconnectionstatechange;
   attribute EventHandler onicegatheringstatechange;
   attribute EventHandler onconnectionstatechange;
+
+  // Legacy Interface Extensions
+  // Supporting the methods in this section is optional.
+  // However, if these methods are supported
+  // it is mandatory to implement according to how they are specified .
```

Comment:
Suggestion: Add a link to the legacy section.
Language suggestion: "However, if these methods are supported, they MUST be implemented
as defined here". Or just delete the "However" - it is implicit in the fact that they're specified, I think.